kv/kv.go

```diff
@@ -176,6 +176,8 @@ type Driver interface {
 type Storage interface {
 	// Begin transaction
 	Begin() (Transaction, error)
+	// Begin transaction with startTs
+	BeginWithStartTs(startTs uint64) (Transaction, error)
```

Comment:
I thought the history read feature is based on creating a snapshot at some user defined TS. Can this case handle it in a similar way?